tests/test_transform_fields.py

```diff
@@ -0,0 +1,114 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+
+"""Test fields in TOML [transform]."""
+import copy
+import unittest
+from pathlib import Path
+from textwrap import dedent
+
+import pytoml
+
+from detection_rules.devtools import guide_plugin_convert_
+from detection_rules.rule import TOMLRule, TOMLRuleContents
+from detection_rules.rule_loader import RuleCollection
+
+RULES_DIR = Path(__file__).parent.parent / 'rules'
+
+
+class TestGuideMarkdownPlugins(unittest.TestCase):
```

Comment:
Are these new test going to fail on older stacks? Should we add an exception just for 8.5+